trap_posix.go

```diff
@@ -11,15 +11,18 @@ import (
 
 func (d *Daemon) trapSignals() {
 	ch := make(chan os.Signal, 1)
-	signal.Notify(ch, syscall.SIGUSR1)
+	signal.Notify(ch, syscall.SIGUSR1, syscall.SIGINT, syscall.SIGTERM, syscall.SIGABRT)
```

Comment:
I don't think we should catch `SIGABRT`; abort means exit immediately and dump core.

Reply:
Removed